src/utils/ErrorHandler.ts

```diff
@@ -18,7 +18,19 @@ export function handleUnknownError(e: unknown, ws: WorkerScript | null = null, i
     e = ws ? basicErrorMessage(ws, msg, "SYNTAX") : `SYNTAX ERROR:\n\n${msg}`;
   } else if (e instanceof Error) {
     // Ignore any cancellation errors from Monaco that get here
-    if (e.name === "Canceled" && e.message === "Canceled") return;
+    if (e.name === "Canceled" && e.message === "Canceled") {
+      return;
+    }
+    if (ws) {
+      console.error(`An error was thrown in your script. Hostname: ${ws.hostname}, script name: ${ws.name}.`);
+    }
+    /**
+     * If e is an instance of Error, we print it to the console. This is especially useful when debugging a TypeScript
+     * script. The stack trace in the error popup contains only the trace of the transpiled code. Even with a source
```

Comment:
The situation with the transpiled trace is particularly unsatisfying right now because it's showing transpiled line numbers, but the original source name. This discrepancy is the biggest issue IMO; I think it will lead to lots of confusion for people using TS/TSX. If the source file names were `.js` or `.transformedJs` or something to make it obvious that the line numbers aren't the same, it would be OK.